src/main/java/com/solvd/pages/mobile/admin/AddJobPageMobile.java

```diff
@@ -0,0 +1,25 @@
+package com.solvd.pages.mobile.admin;
+
+import com.solvd.pages.common.admin.AddJobPageBase;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+
+public  class AddJobPageMobile extends AddJobPageBase {
```

Comment:
did you want to make this class abstract? just asking, since you didn't add device type and the space between public and class is big and it looks similar to your other abstract clasess